packages/babylon-ts-sdk/docs/api/clients.md

```
@@ -278,6 +278,33 @@ Defined in: [packages/babylon-ts-sdk/src/tbv/core/clients/eth/protocol-params-re
+###### Returns
+
+`Promise`\<`bigint`\>
+
+###### Throws
+
+If the deployment does not expose `peginActivationDelay()`, or
+  the decoded payload is not a `bigint`.
+
+###### Implementation of
+
+[`ProtocolParamsReader`](#protocolparamsreader).[`getPeginActivationDelay`](#getpeginactivationdelay-2)
```

markdownlint-cli2 (0.23.2)

[warning] Link fragments should be valid

(MD051, link-fragments)